devel/libecl/include/ert/ecl/ecl_grid.h

```diff
@@ -58,13 +58,21 @@ extern "C" {
 
   double          ecl_grid_get_cell_dx1( const ecl_grid_type * grid , int global_index );
   double          ecl_grid_get_cell_dx3( const ecl_grid_type * grid , int i , int j , int k);
+  double          ecl_grid_get_cell_dx1A( const ecl_grid_type * grid , int active_index);
   double          ecl_grid_get_cell_dy1( const ecl_grid_type * grid , int global_index );
   double          ecl_grid_get_cell_dy3( const ecl_grid_type * grid , int i , int j , int k);
+  double          ecl_grid_get_cell_dy1A( const ecl_grid_type * grid , int active_index);
+  double          ecl_grid_get_cell_dz3( const ecl_grid_type * grid , int i , int j , int k);
+  double          ecl_grid_get_cell_dz1( const ecl_grid_type * grid , int global_index );
+  double          ecl_grid_get_cell_dz1A( const ecl_grid_type * grid , int active_index );
+
```

Comment:
`dz3` comes before `dz1`.  Change order?  What about grouping depending on arguments instead of coordinate?  E.g.

```c++
double ecl_grid_get_cell_dx1(  const ecl_grid_type * grid , int global_index );
double ecl_grid_get_cell_dy1(  const ecl_grid_type * grid , int global_index );
double ecl_grid_get_cell_dz1(  const ecl_grid_type * grid , int global_index );

double ecl_grid_get_cell_dx1A( const ecl_grid_type * grid , int active_index);
double ecl_grid_get_cell_dy1A( const ecl_grid_type * grid , int active_index);
double ecl_grid_get_cell_dz1A( const ecl_grid_type * grid , int active_index );

double ecl_grid_get_cell_dx3(  const ecl_grid_type * grid , int i , int j , int k);
double ecl_grid_get_cell_dy3(  const ecl_grid_type * grid , int i , int j , int k);
double ecl_grid_get_cell_dz3(  const ecl_grid_type * grid , int i , int j , int k);
```